test/lib/mutex/release.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const Utils = require('../../test_lib/utils.js');
+
+const Mutex = artifacts.require('TestMutex');
+
+contract('Mutex.release()', async (accounts) => {
+
+    it('should release lock for an address', async () => {
+
+        let mutex = await Mutex.new();
+
+        let address = accounts[0];
+        await mutex.acquireExternal(address);
+
+        let result = await mutex.releaseExternal.call(address);
+        assert.strictEqual(
+            result,
+            true,
+            'Lock acquire should success.'
+        );
+        await mutex.releaseExternal(address);
+
+        await Utils.expectRevert(
```

Comment:
Same as above. Plus: acquiring should work again after it is released. You can also test it that way.